requirements/requirements.txt

```
@@ -1,6 +1,7 @@
 numpy~=1.16
 pandas>=1.0,<3
-pydantic~=1.7
+pydantic~=2.3
```

Could we have:
```suggestion
pydantic~=2.1
```
in order to support pytorch lightning which is required by autogluon?


```
gluonts 0.13.8+gdd9383d3a1 depends on pydantic~=2.3
pydantic-settings 2.0.3 depends on pydantic>=2.0.1
fastapi 0.103.2 depends on pydantic!=1.8, !=1.8.1, !=2.0.0, !=2.0.1, !=2.1.0, <3.0.0 and >=1.7.4
lightning 2.0.9.post0 depends on pydantic<2.2.0 and >=1.7.4
```